test/utils.test.js

```diff
@@ -1,6 +1,9 @@
 'use strict';
 
-beforeEach(() => jest.resetModules());
+beforeEach(() => {
+    jest.resetModules();
+    jest.setTimeout(20000);
```

Comment:
really?

Reply:
If your adding it to every single test, should just add it to a shared filed referenced by `setupFiles`

Reply:
removed it, didn't need it. might have got there during some angry debugging ;)